Services/Common/OJS.Workers/OJS.Workers.ExecutionStrategies/Sql/PostgreSql/BasePostgreSqlExecutionStrategy.cs

```diff
@@ -254,15 +257,21 @@ private void EnsureDatabaseIsSetup()
 
         private async Task<IDbConnection> CreateConnection()
         {
-            await using var connection = new NpgsqlConnection(this.workerDbConnectionString);
+            var connection = new NpgsqlConnection(this.workerDbConnectionString);
             await connection.OpenAsync();
-            connection.Disposed += (sender, args) =>
```

Comment:
Don't we need this as well in addition to your changes?

Reply:
No, the initial error was that the Disposed property should not be used but StateChange instead. Seems to work the same way.